### PR TITLE
mmc fix

### PR DESCRIPTION
**Pull request by michal-mazurek**
connect https://github.com/resin-os/resinos/issues/88

### CODE REVIEW

**Comment by michal-mazurek** [comment]
@agherzan @floion @telphan

File: layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/fix-mmc-bbb.bb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+DESCRIPTION = "Apply memory pressure fix"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
```
Comment (agherzan) [inline layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/fix-mmc-bbb.bb]: You don't deploy any files in this directory so I suspect it being useless - unless there is something I miss.

File: layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/fix-mmc-bbb.bb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+SRC_URI_append = " file://fix-mmc-bbb.conf"
```
Comment (agherzan) [inline layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/fix-mmc-bbb.bb]: No need to use an append here.

File: layers/meta-resin-beaglebone/recipes-core/images/resin-image.bbappend [hunk 3]
```diff
@@ -1,4 +1,4 @@
 include resin-image.inc
 
 # Overlays for capemgr
```
Comment (agherzan) [inline layers/meta-resin-beaglebone/recipes-core/images/resin-image.bbappend]: This comment doesn't apply anymore

**Pull request merged**